content/en/integrations/guide/aws-integration-with-terraform.md

```diff
@@ -0,0 +1,198 @@
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type = "AWS"
+      identifiers = ["arn:aws:iam::464622532012:root"]
+    }
+    condition {
+      test = "StringEquals"
+      variable = "sts:ExternalId"
+
+      values = [
+        "${var.datadog_aws_integration_external_id}"
+      ]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "datadog_aws_integration" {
+  statement {
+    actions = [<AWS_PERMISSIONS_LIST>]
+
+    resources = ["*"]
+  }
+}
+
+resource "aws_iam_policy" "datadog_aws_integration" {
+  name = "DatadogAWSIntegrationPolicy"
+  policy = "${data.aws_iam_policy_document.datadog_aws_integration.json}"
+}
+
+resource "aws_iam_role" "datadog_aws_integration" {
+  name = "DatadogAWSIntegrationRole"
+  description = "Role for Datadog AWS Integration"
+  assume_role_policy = "${data.aws_iam_policy_document.datadog_aws_integration_assume_role.json}"
+}
+
+resource "aws_iam_role_policy_attachment" "datadog_aws_integration" {
+  role = "${aws_iam_role.datadog_aws_integration.name}"
+  policy_arn = "${aws_iam_policy.datadog_aws_integration.arn}"
+}
+{{< /code-block >}}
+
+## Create resources
+
+Use templates to create and manage the Datadog - Amazon Web Services integration.
+
+### Datadog - Amazon Web Services integration
+
+Provides a Datadog - Amazon Web Services integration resource. This can be used to create and manage the Datadog - Amazon Web Services integration.
+
+#### Example usage
+
+{{< code-block lang="hcl" filename="" disable_copy="false" collapsible="true" >}}
+resource "datadog_integration_aws" "sandbox" {
+  account_id  = "1234567890"
+  role_name   = "DatadogAWSIntegrationRole"
+  filter_tags = ["key:value"]
+  host_tags   = ["key:value", "key2:value2"]
+  account_specific_namespace_rules = {
+    auto_scaling = false
+    opsworks     = false
+  }
+  excluded_regions = ["us-east-1", "us-west-2"]
+}
+{{< /code-block >}}
+Find additional information on the Terraform registry:
+
+- [Schema][7]
+- [Import Command][8] 
+
+### Datadog - Amazon Web Services integration Lambda ARN
+
+Provides a Datadog - Amazon Web Services integration Lambda ARN resource. This can be used to create and manage the log collection Lambdas for an account.
+Update operations are currently not supported with the Datadog API so any change forces a new resource.
+
+#### Example usage
+
+{{< code-block lang="hcl" filename="" disable_copy="false" collapsible="true" >}}
+resource "datadog_integration_aws_lambda_arn" "main_collector" {
+  account_id = "1234567890"
+  lambda_arn = "arn:aws:lambda:us-east-1:1234567890:function:datadog-forwarder-Forwarder"
+}
+{{< /code-block >}}
+Find additional information on the Terraform registry:
+
+- [Schema][9]
+- [Import Command][10] 
+
+### Datadog - Amazon Web Services integration log collection
+
+Provides a Datadog - Amazon Web Services integration log collection resource. This can be used to manage which AWS services logs are collected from for an account.
```

Comment:
```suggestion
Provides a Datadog - Amazon Web Services integration log collection resource. This can be used to manage which AWS services logs are collected from an account.
```